core/store/models/job_spec_v2.go

```diff
@@ -57,7 +57,7 @@ type (
 	OffchainReportingOracleSpec struct {
 		IDEmbed
 		ContractAddress                        EIP55Address   `json:"contractAddress" toml:"contractAddress"`
-		P2PPeerID                              PeerID         `json:"p2pPeerID" toml:"p2pPeerID"         gorm:"column:p2p_peer_id"`
+		P2PPeerID                              PeerID         `json:"p2pPeerID" toml:"p2pPeerID" gorm:"column:p2p_peer_id;default:null"`
```

Comment:
Does this do anything?

Reply:
yeah we needed this, otherwise an unset PeerID would marshal to the db as an empty string and fail the FK constraint

Reply:
Got it